index.js

```diff
@@ -319,10 +425,27 @@ Color.prototype = {
 	},
 
 	clone: function () {
-		return new Color(this.rgb());
+		return createNewColorInstance(cloneValues(this.values));
 	}
 };
 
+Color.prototype.setChannel = function (space, index, val) {
```

Comment:
Could you explain why this method needed to be introduced? I don't feel like it's going to help much of anything.